src/components/manage/Tiles/Hero/Edit.jsx

```diff
@@ -0,0 +1,375 @@
+      });
+    }
+  }
+
+  /**
+   * Change Title handler
+   * @method onChangeTitle
+   * @param {object} titleEditorState Editor state.
+   * @returns {undefined}
+   */
+  onChangeTitle(titleEditorState) {
+    this.setState({ titleEditorState }, () => {
+      this.props.onChangeTile(this.props.tile, {
+        ...this.props.data,
+        title: titleEditorState.getCurrentContent().getPlainText(),
+      });
+    });
+  }
+
+  /**
+   * Change BoldTitle handler
+   * @method onChangeBoldTitle
+   * @param {object} boldTitleEditorState Editor state.
+   * @returns {undefined}
+   */
+  onChangeBoldTitle(boldTitleEditorState) {
+    this.setState({ boldTitleEditorState }, () => {
+      this.props.onChangeTile(this.props.tile, {
+        ...this.props.data,
+        boldTitle: boldTitleEditorState.getCurrentContent().getPlainText(),
+      });
+    });
+  }
+
+  /**
+   * Change Description handler
+   * @method onChangeDescription
+   * @param {object} descriptionEditorState Editor state.
+   * @returns {undefined}
+   */
+  onChangeDescription(descriptionEditorState) {
+    this.setState({ descriptionEditorState }, () => {
+      this.props.onChangeTile(this.props.tile, {
+        ...this.props.data,
+        description: descriptionEditorState.getCurrentContent().getPlainText(),
+      });
+    });
+  }
+
+  /**
+   * Upload image handler
+   * @method onUploadImage
+   * @returns {undefined}
+   */
+  onUploadImage({ target }) {
+    const file = target.files[0];
+    this.setState({
+      uploading: true,
+    });
+    readAsDataURL(file).then(data => {
+      const fields = data.match(/^data:(.*);(.*),(.*)$/);
+      this.props.createContent(getBaseUrl(this.props.pathname), {
+        '@type': 'Image',
+        image: {
+          data: fields[3],
+          encoding: fields[2],
+          'content-type': fields[1],
+          filename: file.name,
+        },
+      });
+    });
+  }
+
+  /**
+   * Render method.
+   * @method render
+   * @returns {string} Markup for the component.
+   */
+  render() {
+    if (__SERVER__) {
+      return <div />;
+    }
+    return (
+      <div
+        onClick={() => this.props.onSelectTile(this.props.tile)}
+        className={['tile', 'hero', this.props.selected && 'selected']
+          .filter(e => !!e)
+          .join(' ')}
+      >
+        {this.props.selected &&
+          !!this.props.data.url && (
+            <div className="toolbar clear-icon">
+              <Button.Group>
+                <Button
+                  icon
+                  basic
+                  onClick={() =>
+                    this.props.onChangeTile(this.props.tile, {
+                      ...this.props.data,
+                      url: '',
+                    })
+                  }
+                >
+                  <Icon name={clearSVG} size="24px" color="#e40166" />
+                </Button>
+              </Button.Group>
+            </div>
+          )}
+        <div className="product-hero">
```

Comment:
"product-hero" comes from Zeelandia project, please use a more descriptive class. We can think in a broader solution, like `tile-inner-wrapper`?

Reply:
Hm. I created and manually amend changes on it . I will change the same.